<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaHistoricooLocacoes</class>
 <widget class="QDialog" name="TelaHistoricooLocacoes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>451</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetListagem">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>811</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CPF CLIENTE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PLACA VEÍCULO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SITUAÇÃO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATA LOCAÇÃO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATA DEVOLUÇÃO</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>VALOR TOTAL</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelCliente">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TODOS CLIENTES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF CLIENTE:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxcPF">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFiltrar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>FILTRAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRetirarFiltro">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>RETIRAR FILTRO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVoltar">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
